src/app/shared/components/cluster-summary/style.scss

```diff
@@ -46,6 +46,13 @@ $steps: 5;
   }
 }
 
+.km-wizard-dialog-summary {
+  .header {
+    font-size: variables.$font-size-subhead-lg;
+    margin-bottom: 16px;
+  }
+}
```

Comment:
This code is the same as above. Instead you could only extract the `@include counter-x;` part to separate rule.